api/src/main/java/uk/gov/dhsc/htbhf/claimant/reporting/payload/ReportClaimPropertiesFactory.java

```diff
@@ -27,12 +31,23 @@ public ReportClaimPropertiesFactory(@Value("${google-analytics.tracking-id}") St
     public Map<String, String> createReportPropertiesForClaimEvent(ReportClaimMessageContext context) {
         Map<String, String> reportProperties = new LinkedHashMap<>();
         reportProperties.putAll(mapValuesToString(createMandatoryPropertiesMap()));
-        reportProperties.putAll(mapValuesToString(createEventPropertiesMap(context, CLAIM, 0)));
+        String eventValue = getEventValue(context.getUpdatedClaimFields());
+        reportProperties.putAll(mapValuesToString(createEventPropertiesMap(context, CLAIM, eventValue)));
         reportProperties.putAll(mapValuesToString(createCustomDimensionMap(context)));
         reportProperties.putAll(mapValuesToString(createCustomMetricMapForClaimEvent(context)));
         return reportProperties;
     }
 
+    private String getEventValue(List<UpdatableClaimantField> updatedClaimFields) {
+        return isEmpty(updatedClaimFields)
```

Comment:
I may have made a mistake in the description. EventValue is a metric - it needs to be an integer. We should be using event label for this.

Reply:
Updated to switch around the value and label to the correct values.